wcomponents-core/src/main/java/com/github/bordertech/wcomponents/util/XMLUtil.java

```diff
@@ -24,18 +24,7 @@
 	/**
 	 * XML Declaration.
 	 */
-	public static final String XML_DECLERATION = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>";

```

Comment:
This is technically an API change.

Might need to keep XML_DECLERATION and mark as deprecated.